### PR TITLE
Probably fixes that ancient issue with roundstart borgs not getting malf/traitor law from the AI.

### PR DESCRIPTION
**Pull request by Intigracy**
also qdel everywhere

### CODE REVIEW

File: code/game/gamemodes/gameticker.dm [hunk 1]
```diff
@@ -309,12 +309,6 @@ var/global/datum/controller/gameticker/ticker
 					flick("station_explode_fade_red",cinematic)
 					world << sound('sound/effects/explosionfar.ogg')
 					cinematic.icon_state = "summary_malf"
-				if("blob") //Station nuked (nuke,explosion,summary)
-					flick("intro_nuke",cinematic)
-					sleep(35)
-					flick("station_explode_fade_red",cinematic)
-					world << sound('sound/effects/explosionfar.ogg')
-					cinematic.icon_state = "summary_selfdes"
```
Comment (Intigracy) [inline code/game/gamemodes/gameticker.dm]: Removed because the else under it is the exact same code.

File: code/game/gamemodes/gameticker.dm [hunk 2]
```diff
@@ -134,7 +134,7 @@ var/global/datum/controller/gameticker/ticker
 		src.mode = config.pick_mode(master_mode)
 	if (!src.mode.can_start())
 		world << "<B>Unable to start [mode.name].</B> Not enough players, [mode.required_players] players needed. Reverting to pre-game lobby."
-		del(mode)
+		qdel(mode)
```
Comment (N3X15) [inline code/game/gamemodes/gameticker.dm]: Don't qdel /datums, it causes problems.

File: code/modules/admin/verbs/randomverbs.dm [hunk 3]
```diff
@@ -524,14 +524,14 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
-					if(new_character.mind.special_role=="traitor")
-						call(/datum/game_mode/proc/add_law_zero)(new_character)
 				if("AI")
 					new_character = new_character.AIize()
 					if(new_character.mind.special_role=="traitor")
 						call(/datum/game_mode/proc/add_law_zero)(new_character)
+				if("Cyborg")//More rigging to make em' work and check if they're traitor.
+					new_character = new_character.Robotize()
+					if(new_character.mind.special_role=="traitor")
+						call(/datum/game_mode/proc/add_law_zero)(new_character)
```
Comment (N3X15) [inline code/modules/admin/verbs/randomverbs.dm]: You didn't need to move this.

**Comment by Intigracy** [comment]
Yeah I realized after the fact.

No harm in it anyway.

File: code/modules/admin/verbs/randomverbs.dm (hunk 3 above)
Comment (Intigracy) [inline code/modules/admin/verbs/randomverbs.dm]: Yeah I realized after the fact.

No harm in it anyway.

**Pull request merged**